pages/services/cassandra/2.4.0-3.0.16/operations/index.md

```diff
@@ -132,7 +132,7 @@ dcos {{ model.packageName }} --name=<service-name> plan start backup-s3 \
     -p S3_BUCKET_NAME=$S3_BUCKET_NAME
     -p HTTPS_PROXY=http://internal.proxy:8080
 ```
-It may also be necessary to set the `AWS_SESSION_ID`, depending on how you authenticate with AWS.
+It may also be necessary to set the `AWS_SESSION_ID`, `AWS_SESSION_TOKEN` depending on how you authenticate with AWS.
```

Comment:
```suggestion
It may also be necessary to set the `AWS_SESSION_ID` or `AWS_SESSION_TOKEN` depending on how you authenticate with AWS.
```
```suggestion
It may also be necessary to set the `AWS_SESSION_ID`, `AWS_SESSION_TOKEN` depending on how you authenticate with AWS.
```